ios/RNTextSize.m

```diff
@@ -187,7 +178,9 @@ - (dispatch_queue_t)methodQueue {
   [textStorage addLayoutManager:layoutManager];
 
   NSMutableArray<NSNumber *> *result = [[NSMutableArray alloc] initWithCapacity:texts.count];
-  const CGFloat epsilon = 0.001;
+
+  const CGFloat scaleMultiplier = _bridge ? _bridge.accessibilityManager.multiplier : 1.0;
```

Comment:
It looks like this logic is shared between both here and later. Can you extract it out into its own function?

```objc
- (CGFloat)fontScaleMultiplier {
  return _bridge ? _bridge.accessibilityManager.multiplier : 1.0;
}
```